CHANGELOG.md

```diff
@@ -70,6 +70,42 @@ $govuk-suppressed-warnings: (
 
 This was added in [#2911 Add warning suppression functionality](https://github.com/alphagov/govuk-frontend/pull/2911)
 
+#### Configure components in JavaScript
+
+JavaScript components receiving configuration through data attributes now accept the same configuration options when instantiated in JavaScript. These components are:
```

Comment:
Is there a plainer option for 'instantiated' that is still accurate? Would something like this be accurate in any way? - ' ...the same configuration options as when an instance is created in JavaScript.'

Reply:
Or maybe we can turn this sentence around a bit to put a different emphasise? 
'JavaScript components can now get the same configuration options in 2 ways - through data attributes and when creating an instance.'

Reply:
That second sentence feels more direct, I like it 😄 It muddies the fact that the data attributes were already there and that the change is about receiving configuration in JavaScript, though. Would it hurt to add some timing precisions in the second part:

> JavaScript components can ~~now~~ get the same configuration options in 2 ways - through data attributes, **as before,** and **now** when creating an instance.

Reply:
Yup, that works. Fab 😄